### PR TITLE
Added Feature for empty variable name

### LINKED ISSUE #930: Combinational Analysis Bugs
**Describe the bug**

1. When we enter no value in the dialog Box then too , it takes us to the table with empty fields.
2. When there is a comma (delimeter) left by mistake then it creates another empty column in the combinational table. ( #291 is similar but not yet reflected in website )

**To Reproduce**
1. Go to 'circuitverse.org'
2. Click on 'simulator'
3. Go to 'Tools' => Combinational Analysis

**Expected behavior**
It should not take empty values forward to create a table . This may create a confusion while creating the circuit. 

**Screenshots**
1.
Issue4_1
2.
Issue4_2

**Desktop (please complete the following information):**
 - OS: [Windows 10]
 - Browser [Chrome]

**Are you working on this issue? (Yes/No)**
Yes

### PR DESCRIPTION
**Pull request by Shivansh2407**
Fixes #930 

#### Describe the changes you have made in this pr -

- [x] I have added a Alert so whenever the user misses out entering the input / output variable , then the alert pops out. 

- [x] Also I have remove empty values from the input and output Lists.

### Screenshots of the changes (If any) -
1.
patch_20_1

2.
patch_20_2

### CODE REVIEW

**Comment by Shivansh2407** [comment]
@satu0king Please review this and let me know 👍

**Comment by Shivansh2407** [comment]
@tachyons please review 👍

**Comment by Shivansh2407** [comment]
@tachyons and @nitin10s review please 👍

**Comment by Shivansh2407** [comment]
@satu0king , @tachyons Review Please 👍

File: public/js/combinationalAnalysis.js [hunk 1]
```diff
@@ -16,16 +16,24 @@ createCombinationalAnalysisPrompt=function(scope=globalScope){
...
           click: function() {
             // //console.log($("#inputNameList"),$("#inputNameList").val(),$("#inputNameList").html());
-            var inputList=$("#inputNameList").val().split(',');
-            var outputList=$("#outputNameList").val().split(',');
+            var inputList = [] ;
+            var outputList = [] ;
+            inputList=$("#inputNameList").val().split(',');
+            outputList=$("#outputNameList").val().split(',');
             inputList = inputList.map( x => x.trim() );
+            inputList = inputList.filter(function(e){return e});
```
Comment (satu0king) [inline public/js/combinationalAnalysis.js]: What does this filter do ?

**Comment by satu0king** [comment]
Merged by #1284

**Pull request closed without merging**